micropub.php

```diff
@@ -114,17 +110,94 @@ public static function parse_query( $wp ) {
 		else {
 			$user_id = Micropub::authorize();
 		}
+		// If q is set this must be a query so 
+		if ( isset( $_GET['q'] ) ) {
+			Micropub::query_handler( $user_id );  // calls wp_die
+			wp_die();
```

Comment:
you can remove this, since `query_handler` calls it. and probably put `form_handler` in an `else`.